packages/eigen/eigen.0.1.0/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+maintainer: "Liang Wang <ryanrhymes@gmail.com>"
+authors: [ "Liang Wang" ]
+license: "MIT"
+homepage: "https://github.com/owlbarn/eigen"
+dev-repo: "git+https://github.com/owlbarn/eigen.git"
+bug-reports: "https://github.com/owlbarn/eigen/issues"
+doc: "https://owlbarn.github.io/eigen/eigen"
+build: [
+  ["dune" "build" "eigen_cpp/libeigen_cpp_stubs.a"]
+  ["dune" "build" "-p" name]
```

Comment:
```suggestion
  ["dune" "build" "-p" name "-j" jobs]
```

Reply:
I hope this will not break the build. C stubs in presence of external libraries are still weak when it comes to concurrent builds